app/src/main/java/edu/umd/cmsc436/mstestsuite/MainPresenter.java

```diff
@@ -120,13 +125,13 @@ public void onBottomSheetStateChange(int newState) {
     }
 
     @Override
-    public boolean onBackPressed() {
+    public void onBackPressed() {
```

Comment:
This method returns true if the activity should call the "real" onBackPressed or not, and false if it just switches the adapter, and should be left alone